patches/chrome-browser-resources-tools-optimize_webui.gni.patch

```diff
@@ -1,12 +1,13 @@
 diff --git a/chrome/browser/resources/tools/optimize_webui.gni b/chrome/browser/resources/tools/optimize_webui.gni
-index d3461400579c1ee232b2561805fafb1a4a3dba25..4757139a5c0ad526544ceda15ec0a84caa0ad30e 100644
+index 25732ec00e575af482bfe95cd479633679eb0b16..33eb8312bd8c5c772107677b9143fe4998eba1a0 100644
 --- a/chrome/browser/resources/tools/optimize_webui.gni
 +++ b/chrome/browser/resources/tools/optimize_webui.gni
-@@ -56,6 +56,7 @@ template("optimize_webui") {
-       "//resources/polymer/v3_0/|$polymer_path",
+@@ -70,6 +70,8 @@ template("optimize_webui") {
        "//resources/|$resources_path",
      ]
+ 
 +    import("//brave/browser/resources/tools/brave_optimize_webui.gni") deps += brave_webui_deps external_paths += brave_webui_external_paths
++
```

Comment:
same as above

Reply:
Fixed in e28b377e795a2cd34acebd898845cd451478167a.